### ISSUE TITLE
"Scaled" coordinates

### ISSUE THREAD

**Issue by jburel** [ome/omero-iviewer#251]
By default, OMERO iviewer indicates X and Y coordinates of the mouse pointer in terms of pixels.

Given that the image data holds the scaling factor, once you set a reference point in an image by giving X, Y and Z coordinates (in terms of a mouse brain atlas [http://chamber.mrc.ox.ac.uk/iviewer/?dataset=51], they can actually represent anteroposterior, dorsoventral, and mediolateral for example), then OMERO can work out the X, Y, Z coordinates in µm or mm in the real world. This would be helpful in some use cases I believe.

For example, ImageJ supports this kind of scaling, though it does not allow you to choose arbitrary reference point.

**Comment by jburel** [comment]
Thanks for the suggestion,
how do you see that working? Indicates the "scaled" coordinates alongside the coordinates in pixels or having the ability to toggle between the two views.

**Comment by kouichi-c-nakamura** [comment]
> how do you see that working? Indicates the "scaled" coordinates alongside the coordinates in pixels or having the ability to toggle between the two views.

Good point. Either is fine. But I'd prefer toggling because it would look less busy on the screen.

To get it work, somewhere in the image metadata, you'll need to define the reference point at X1, Y1, Z1 (possibly T1 as well) eg. [732, 500, 1] in pixels and you assign X, Y, and Z values in the scale of real world, eg. [-2.36, 4.51, 1.52] in mm.

**Comment by kouichi-c-nakamura** [comment]
I've found that Human Brain Project web site has this feature implemented.
http://cmbn-navigator.uio.no/navigator/filmstripzoom/filmstripzoom.html?atlas=200000&series=222&preview=WHSRatv2Preview.png

From the pulldown menu at the top  left, you can choose "Bregma (mm)" and that is exactly what I'm talking about. The cursor positions are shown as X, Y coordinates from the bregma (a reference point at the surface of the skull).
